Examples/VisionExamples/Examples/Basics/GettingStartedViewController.swift

```diff
@@ -0,0 +1,87 @@
+/**
+ This file is used in a public documentation to show basic configuration steps.
+ */
+
+// Need to include Vision functionality
+import MapboxVision
+// Need to include Vision AR functionality
+import MapboxVisionAR
+// Need to include Vision Safety functionality
+import MapboxVisionSafety
+
+class GettingStartedViewController: UIViewController {
+    
+    // MARK: - Private properties
+    
+    private var videoSource: CameraVideoSource!
+    
+    private var visionManager: VisionManager!
+    private var visionARManager: VisionARManager!
+    private var visionSafetyManager: VisionSafetyManager!
+    
+    // MARK: - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // create a video source obtaining buffers from camera module
+        videoSource = CameraVideoSource()
+        
+        // create VisionManager with video source
+        visionManager = VisionManager.create(videoSource: videoSource)
+        // setting up the `VisionSafetyManagerDelegate`
```

Comment:
To conform to the common imperative tone. Here and later
```suggestion
        // set up the `VisionSafetyManagerDelegate`
```